src/task/MonitorTask.ts

```diff
@@ -296,11 +298,41 @@ export default class MonitorTask
             Maybe.of(observer.connectionDidBecomePoor).map(f => f.bind(observer)());
           });
         }
-      } else {
+      },
+      () => {
         this.context.audioVideoController.forEachObserver((observer: AudioVideoObserver) => {
           Maybe.of(observer.connectionDidBecomeGood).map(f => f.bind(observer)());
         });
       }
+    );
+
+    this.applyHealthPolicy(
+      this.sendingAudioFailureHealthPolicy,
+      connectionHealthData,
+      () => {
+        this.context.eventController?.publishEvent('sendingAudioFailed');
```

Comment:
Sorry, dint got this point earlier, so for the `receivingAudioDropped` event we also publish `attributes` with it:
- https://github.com/aws/amazon-chime-sdk-js/blob/main/src/task/MonitorTask.ts#L289
- https://github.com/aws/amazon-chime-sdk-js/blob/main/src/task/MonitorTask.ts#L408-L428

I recommend we also include the relevant attributes with both of these new events. Also, we would like to update [this](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/06_Meeting_Events.md#meeting-attributes) section in the Meeting events guide so that builders know which attributes is available under which meeting event.

Reply:
I intentionally left those additional attributes out because the [standard attributes](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/06_Meeting_Events.md#standard-attributes) seemed like they're enough for the `sendingAudioXXX` events and I wanted to be frugal with the data we send. Let me know what you think.

If we decide to add additional attributes I think we can include the following:
- meetingDurationMs
- meetingStartDurationMs
- signalingOpenDurationMs
- iceGatheringDurationMs
- attendeePresenceDurationMs

And leave out maxVideoTileCount and poorConnectionCount. Thoughts?

Reply:
So basically if in situation like these, there are existing meeting events that also trigger which include above attributes then we should be fine to drop the attributes altogether. If not then would be good to add them.

Reply:
As discussed, we cannot rely on existing meeting events so I added the attributes I listed above